### PR TITLE
Added abbreviated printing for units

### LINKED ISSUE #13269: Printing should use short representation of quantities.
There is a test that explicitly expects that printing does not use `abbrev` but `name`:
https://github.com/sympy/sympy/blob/8e962a301d7cc2d6fc3fa83deedd82697a809fd6/sympy/physics/units/tests/test_quantities.py#L87
Is there a reason behind this? I find it quite user-unfriendly to look at `1.34*meter/second` instead of `1.34*m/s`.
It would be very easy to change here: https://github.com/sympy/sympy/blob/8e962a301d7cc2d6fc3fa83deedd82697a809fd6/sympy/printing/str.py#L713
But then, the above test would fail. Is anyone emotionally attached to the current verbose display of units and quantities?

### PR DESCRIPTION
**Pull request by Subhash-Saurabh**
Added `abbrev` parameter which prints units in abbreviated form.
fixes #13269
closes #13310

### CODE REVIEW

**Comment by Subhash-Saurabh** [comment]
@ylemkimon Can you please review this PR.

**Comment by ylemkimon** [comment]
Please update the documentation.

**Comment by Subhash-Saurabh** [comment]
@ylemkimon The current documentation doesn't contains any information about the settings that can be passed as arguments for any printer. Should I add documentation regarding the settings or what other documentation changes do you suggest.

**Comment by ylemkimon** [comment]
I think just adding documentation regarding `abbrev` to the docstring would be okay.

**Review by ylemkimon — APPROVED** [review]

File: sympy/printing/str.py [hunk 1]
```diff
@@ -706,6 +707,8 @@ def _print_Complement(self, expr):
         return r' \ '.join(self._print(set) for set in expr.args)
 
     def _print_Quantity(self, expr):
+        if self._settings.get("abbrev", True):
```
Comment (ylemkimon) [inline sympy/printing/str.py]: The second parameter should be the default value.

**Review by ylemkimon — CHANGES_REQUESTED** [review]

File: sympy/printing/str.py (hunk 1 above)
Comment (Subhash-Saurabh) [reply sympy/printing/str.py]: I made such a blunder still all the test passed. :)

**Review by ylemkimon — APPROVED** [review]

**Pull request merged**